tempesta_fw/connection.h

```diff
@@ -284,6 +289,55 @@ tfw_srv_conn_hasnip(TfwSrvConn *srv_conn)
 	return test_bit(TFW_CONN_B_HASNIP, &srv_conn->flags);
 }
 
+/*
+ * Tell if connection is temporary inactive due to full work queue.
+ */
+static inline bool
+tfw_srv_conn_busy(TfwSrvConn *conn)
+{
+	if (test_bit(TFW_CONN_B_BUSY, &conn->flags))
+		return true;
+	rmb();
```

Comment:
What is the purpose of read memory barrier here?

Reply:
@keshonok Thank you for review!
Due to CPU reordering some requests may be occasionally scheduled on busy connection in the moment of clearing busy bit. To prevent this situation memory barriers are used.
Added comment for `tfw_srv_clear_busy_delay()`.